src/controllers/expenseController.js

```diff
@@ -0,0 +1,94 @@
+const expensesService = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const filteredExpenses = expensesService.getAllExpenses(
+    userId,
+    categories,
+    from,
+    to,
+  );
```

Comment:
It seems you are passing the `userId` directly from the request query to the `getAllExpenses` function. This could lead to potential security issues as a user can pass any `userId` to see the expenses of other users. It would be better to get the `userId` from the authenticated user's session or token.